doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
  */
 ////
 
-calcurse(1)
+CALCURSE(1)
 ===========
 
-Name
+NAME
 ----
 
-calcurse - text-based organizer
+calcurse - terminal-based organizer for interactive and command line use
 
-Synopsis
+SYNOPSIS
 --------
 
-[verse]
-'calcurse' *-Q* [options] [*--from* <date>] [*--to* <date>|*--days* <num>]
-'calcurse' *-G* [options]
-'calcurse' *-i*<file>
-'calcurse' *-x*<format>
-'calcurse' *--gc*
-'calcurse' *--status*
-'calcurse' *--version*
-'calcurse' *--help*
-
-Description
------------
+--
+*calcurse* [-D 'directory'] | [-C 'confdir'] | [-c 'calendar-file']
+
+*calcurse* -Q [*--from* 'date'] [*--to* 'date' | *--days* 'number']
+
+*calcurse* -a | -d 'date' | -d 'number' | -n | -r['number'] | -s['date'] | -t['number']
+
+*calcurse* -h | --status | -G | -g | -i 'file' | -P | -x['file'] | --daemon
+--
 
-Calcurse is a text-based calendar and scheduling application. It helps keeping
-track of events, appointments and everyday tasks.  A configurable notification
-system reminds user of upcoming deadlines, and the curses based interface can
-be customized to suit user needs.  All of the commands are documented within an
-online help system.  
+Briefly, the first form shows how to invoke calcurse interactively; the
+remaining are command line forms. The second form shows queries, the third
+short-form queries; both may be combined with filter options as well as
+format options. The fourth form is miscellaneous help and support functions.
+All details are in <<_options,OPTIONS>>.
 
-Options
+DESCRIPTION
+-----------
+
+Calcurse is a calendar and scheduling application for use in a terminal
```

Comment:
I know that this has just been copied from the previous revision of the file but I wonder whether we should always use lower case for the proper name `calcurse`, even if it is at the beginning of a sentence. Opinions?

Reply:
Corrected.